app/jobs/remote_indexing_job.rb

```diff
@@ -0,0 +1,39 @@
+##
+# job to reindex a DOR object to Solr using the dor_indexing_app endpoint
+class RemoteIndexingJob < GenericJob
+  queue_as :indexing_remote
```

Comment:
that was my intent.

i was under the impression that the queue name just gets used at the time the job is queued, and that the queue doesn't have to be declared ahead of time.  if that's wrong, where else would i have to add the new queue name?  and, i assume i'd also have to change the old `:indexing` queue name to `:indexing_local` in that case?